src/refiners/foundationals/latent_diffusion/auto_encoder.py

```diff
@@ -188,47 +188,95 @@ def __init__(self, device: Device | str | None = None, dtype: DType | None = Non
 
 
 class LatentDiffusionAutoencoder(Chain):
+    """Latent diffusion autoencoder model.
+
+    Attributes:
+        encoder_scale: The encoder scale to use.
+    """
+
     encoder_scale = 0.18125
 
     def __init__(
         self,
         device: Device | str | None = None,
         dtype: DType | None = None,
     ) -> None:
+        """Initializes the model.
+
+        Args:
+            device: The PyTorch device to use.
+            dtype: The PyTorch data type to use.
+        """
         super().__init__(
             Encoder(device=device, dtype=dtype),
             Decoder(device=device, dtype=dtype),
         )
 
     def encode(self, x: Tensor) -> Tensor:
+        """Encode an image.
+
+        Args:
+            x: The image tensor to encode.
+
+        Returns:
+            The encoded tensor.
+        """
         encoder = self[0]
         x = self.encoder_scale * encoder(x)
         return x
 
     def decode(self, x: Tensor) -> Tensor:
+        """Decode a latent tensor.
+
+        Args:
+            x: The latent to decode.
+
+        Returns:
+            The decoded image tensor.
+        """
         decoder = self[1]
         x = decoder(x / self.encoder_scale)
         return x
 
+    # backward-compatibility alias
+    # TODO: deprecate this method
```

Comment:
```suggestion
```

Reply:
This one have been introduced recently in #213: we should keep it (only `decode_latents` was kept for backward compat and should be removed at some point). Am I missing something?